packages/chart/src/components/BarChart.tsx

```diff
@@ -9,6 +9,9 @@ import { BarStackHorizontal } from '@visx/shape'
 
 export default function BarChart({ xScale, yScale, seriesScale, xMax, yMax, getXAxisData, getYAxisData, animatedChart, visible }) {
   const { transformedData: data, colorScale, seriesHighlight, config, formatNumber, updateConfig, colorPalettes, formatDate, parseDate } = useContext<any>(Context)
+
+  //debugger;
```

Comment:
Nit: try to remove large gaps between variables and commented code when possible. Great use of the debugging tools in this file though. Great to see debugging happening outside of console.log statements. Thank you!

Reply:
I usually add those gaps to that later its supposed to jump out at me to remove... in addition to doing a global searchg check.  ANyway removing